setup.py

```diff
@@ -35,7 +35,7 @@
         'monotonic>=1.6',
         'orderedset>=2.0.3',
         'phonenumbers~=8.12.12',
-        'pypdf2>=2.0.0',
+        'pypdf2 < 3.0.0',
```

Comment:
why  'pypdf2 < 3.0.0',  seems like this could install 1.0.0 of that library. Did you mean ~=3.0.0 ?

Reply:
Version 3 has breaking changes.